src/icp_v3.cc

```diff
@@ -76,7 +76,7 @@ ICP3State::created(StoreEntry *e)
     icpCreateAndSend(codeToSend, 0, url, header.reqnum, 0, fd, from, al);
 
     // TODO: StoreClients must either store/lock or abandon found entries.
-    //if (!e->isNull())
+    //if (e)
```

Comment:
This if-statement implies that call to confirmAndPrepHit(*e) is dereferencing a nullptr.

Either this if is unnecessary or there is a missing if(!e) earlier in this function. See how ICP2State::created does it.